mintapi/api.py

```diff
@@ -520,9 +494,9 @@ def get_trend_data(
         date_filter : DateFilter.Options
             date window. must use predefined enum windows or pass a start and end date with CUSTOM enum
         start_date : Optional[str], optional
-            optional start date (YYYY-mm-dd) if using enum CUSTOM, by default None
+            optional start date (mm-dd-yy) if using enum CUSTOM, by default None
```

Comment:
Is this a breaking change?

Reply:
@mrooney it does, but I made a judgment call to ensure that we have consistent date formatting across the board with date inputs. I should've done this when we merged trends in the first place, but didn't. This way, we don't break transaction date input formats here.

Reply:
@burkematthew got it. Is this a format that Mint uses in its output somewhere? That could make sense if so, otherwise I'd say mm-dd-yy is a bit Western-centric / ambiguous and would prefer YYYY-mm-dd.

Reply:
@mrooney from what I have seen, I don't think it is. It's simply what we've used as accepted input historically in the took. That being said, I agree with your point (and typically prefer that date format anyways). However, it would be a breaking change that seems to me would be better suited for a major release. Where things stand now, at least we'd keep consistent date formatting for input.

Reply:
@burkematthew thanks for the explanation! Please excuse me if I'm being dense, but if it is a breaking change no matter what, and we both prefer the YYYY-mm-dd format, wouldn't it make more sense to have a breaking change to a better format instead of a breaking change to a format neither of us prefer? 🤔 If I misunderstood and the change as you have here is not a breaking change, then it makes sense to align now and improve later :)

Reply:
@mrooney no worries! 

The difference is that we'd be breaking a format that has been around for quite some time (at least since I got involved) instead of breaking a format that has been around for a month and probably minimally adopted at this point.

Reply:
Got it, thanks for the context, I think your approach makes sense then. Since the two formats have different numbers of characters, we can later support both by detecting which one it is and document the preferred YYYY-mm-dd one.